### PR TITLE
[TwigBundle] Add missing PHPDoc on FilesystemLoader::findTemplate() about `$throw` parameter

### PR DESCRIPTION
**Pull request by Kevinrob**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php [hunk 1]
```diff
@@ -58,6 +58,7 @@ public function exists($name)
      * Otherwise the template is located using the locator from the twig library.
      *
      * @param string|TemplateReferenceInterface $template The template
+     * @param Boolean $throw This parameter is only here for matching the parent signature
```
Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php]: Should be `bool`, and can you please align the params like this:

```php
@param string|TemplateReferenceInterface $template The template
@param bool                              $throw    This parameter is only here for matching the parent signature
```

Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php]: By the way, why don't we use `{@ineritdoc}` here instead?

**Comment by OskarStark** [comment]
:+1: for inherit

**Comment by Kevinrob** [comment]
@xabbuh I don't think that using `{@ineritdoc}` is a good idea, because it's not the same as parent method. Here we accept `TemplateReferenceInterface` for `$name` too.

And by the way, before Twig 2.0 there isn't any PHPDoc on this method.

File: src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php [hunk 2]
```diff
@@ -58,6 +58,7 @@ public function exists($name)
      * Otherwise the template is located using the locator from the twig library.
      *
      * @param string|TemplateReferenceInterface $template The template
+     * @param bool                              $throw    This parameter is only here for matching the parent signature
```
Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php]: It was omitted on purpose. Like you describe, it's not used.

**Pull request closed without merging**